api/types/network/network.go

```diff
@@ -13,6 +13,13 @@ type IPAM struct {
 	Config  []IPAMConfig
 }
 
+// EndpointOptions represents the network options for endpoint creation
+type EndpointOptions struct {
```

Comment:
its used in cli and opts. I read the readme regarding consumers of the api and cli was one of them. So I parked it here.
However since cli and opts is moved to docker/cli repo I will be moving it there and will only be used in the cli.